### DISCUSSION TITLE
Set user relation to new record

### DISCUSSION THREAD

**Discussion by jtalbourdet** [Budibase/budibase#6146] [Help]
Hello,

I have a company table, a user table and a request table. I use the base budibase. I have a relationship between user and company (1 user for several companies). and for requests, a request is linked to a company.

I want that when a user creates a request, it is automatically linked to his company.

For this I use the action but I can't pass the current user's company relationship to the request.

**Comment by melohagan** [comment]
Hi @jtalbourdet 

There should bindings for all of the Current Users' fields: https://docs.budibase.com/docs/users-table#current-user-bindings

So you should be able to grab the company relationships with something like: `{{ Current User.Companies }}`
This will be an array, which could then be passed into the request that is saved.
